### ISSUE TITLE
Logic app exception

### ISSUE THREAD

**Issue by dean-morgan** [OfficeDev/microsoft-teams-apps-requestateam#250]
The ProcessTeamRequest throws an error at the Create Group action with the following error message:

{
  "error": {
    "code": "Request_BadRequest",
    "message": "The property is missing a required prefix/suffix per your **organization's Group naming requirements.**",
    "innerError": {
      "date": "2021-04-16T18:26:19",
      "request-id": "xxxxxx-xxxx-xxxx-xxxx-f28a1294a3ff",
      "client-request-id": "xxxxxx-xxxx-xxxx-xxxx-f28a1294a3ff"
    },
    "details": [
      {
        "target": "displayName",
        "suggestedPropertyValue": "Deans super site - M365",
        "code": "MissingPrefixSuffix"
      }
    ]
  }
}

**Comment by alexc-MSFT** [comment]
@dean-morgan This is because you have a naming convention at the Azure AD level.

You would have to ensure that the team name the user enters when requesting a team has the required prefix and suffix added as per your naming policy.

It's worth looking at the latest code though as I've now added dedicated naming convention functionality into Request-a-team which you could use alongside your Azure AD naming policy and this should work well.

**Comment by dean-morgan** [comment]
Yes, I figured out that was the case and hard coded the appended data into the string that feeds the Alias. It would be super helpful if the check existence logic app could fill in the suggestedPropertyValue into the name field. That way if there were dynamic naming conventions, like using the department to append, it could manage that for the user since hard coding will fail in that kind of case.

**Comment by LindaGartner** [comment]
@dean-morgan
I have the same issue with a Azure suffix. Can you help telling me where you hard coded the appended data? I'm new in Power apps but would love to use the great functionality in this App. Thanks in advance for your help
